src/plans/details/tabs/Hooks/utils/utils.ts

```diff
@@ -147,9 +153,9 @@ export const onUpdatePlanHooks = async (props: onUpdatePlanHooksProps) => {
     preHookSet,
   } = formData;
 
-  const newPlan = deepCopy(plan);
+  let newPlan = deepCopy(plan);
 
-  await createUpdateOrDeleteHook({
+  newPlan = await createUpdateOrDeleteHook({
```

Comment:
i can see
plan: newPlan,
And u overriding newPlan? 
why setting newPlan? u dont return it

Reply:
After off-line discussion, I fixed the code by removing the `let` var declaration.